evap/evaluation/tools.py

```diff
@@ -16,7 +16,11 @@
 from django.utils.datastructures import MultiValueDict
 from django.utils.translation import get_language
 from django.views.generic import FormView
-from django_stubs_ext import StrOrPromise
+
+if TYPE_CHECKING:
+    from django_stubs_ext import StrOrPromise
+else:
+    StrOrPromise = Any
```

Comment:
@niklasmohrin this means that at test runtime, even if `StrOrPromise` would be available with the correct type, we will have it being equal to `Any`. Run-Time Type Checking (typeguard) thus wouldn't catch violations. Maybe it's better to use
```suggestion
try:
    from django_stubs_ext import StrOrPromise
except ImportError:
    assert not TYPE_CHECKING
    StrOrPromise = Any
```
?

Reply:
Thats better, yes